evm/Dockerfile

```
@@ -34,7 +38,7 @@ RUN apt-get update -y &&\
 
 # Copy the World EVM binary, start script, and configuration files
 COPY --from=build /go/bin/world-evm /usr/local/bin/
-COPY scripts/start-sequencer.sh scripts/app.toml scripts/config.toml ./
+COPY evm/scripts/start-sequencer.sh evm/scripts/app.toml evm/scripts/config.toml ./
```

Hadolint

[warning] `COPY` to a relative destination without `WORKDIR` set.

(DL3045)